src/main/cms/cms_menu_imu.c

```diff
@@ -391,6 +391,73 @@ static CMS_Menu cmsx_menuFilterPerProfile = {
     .entries = cmsx_menuFilterPerProfileEntries,
 };
 
+static uint8_t cmsx_dstPidProfile;
```

Comment:
@AndersHoglund True, I had to think twice to figure out how the condition works. Turns out that `DISABLE_EXTENDED_CMS_MENUS` is defined when a target needs to free some space. Changing this PR, than also PR #3601 needs to be changed to stay consistant.